lib/web_ui/dev/test_runner.dart

```diff
@@ -186,6 +181,20 @@ class TestCommand extends Command<bool> with ArgUtils {
     return true;
   }
 
+  /// Preparations before running the tests such as booting simulators or
+  /// creating directories.
+  Future<void> _prepare() async {
+    if(!environment.webUiTestResultsDirectory.existsSync()) {
```

Comment:
Add one space between `if` and `(`.

Reply:
:D how come I didn't run the format on this one! Thanks for noticing.